src/persist-client/src/cli/admin.rs

```diff
@@ -671,7 +671,7 @@ async fn force_gc(
 /// Exposed for `mz-catalog`.
 pub const CATALOG_FORCE_COMPACTION_FUEL: Config<usize> = Config::new(
     "persist_catalog_force_compaction_fuel",
-    0,
+    131_072,
```

Comment:
This might also make us slower, but set in production.

Reply:
@bkirwi can you weigh in on this one?

Reply:
I'll cc @jkosh44 as well, but personally I don't think it's super valuable to set this to a highish value indefinitely, especially for new envs that were never affected by the issue that we added it for. I'd personally leave this one alone...

Reply:
It might be worth setting it to a low non-zero value like 1024 though so we can ensure that the catalog shard remains compacted. Or do you think it's unlikely that the shard will become severely uncompacted again?

Reply:
Whatever we set it to we should also update the launch darkly flag to match because in prod it's currently sitting at 131,072.